great_expectations/util.py

```diff
@@ -1723,33 +1724,41 @@ def get_context(
         DataContext,
     )
 
-    if (
-        CloudDataContext.is_ge_cloud_config_available(
-            ge_cloud_base_url=ge_cloud_base_url,
-            ge_cloud_access_token=ge_cloud_access_token,
-            ge_cloud_organization_id=ge_cloud_organization_id,
-        )
-        or ge_cloud_mode
-    ):
-        return CloudDataContext(
-            project_config=project_config,
-            runtime_environment=runtime_environment,
-            context_root_dir=context_root_dir,
-            ge_cloud_base_url=ge_cloud_base_url,
-            ge_cloud_access_token=ge_cloud_access_token,
-            ge_cloud_organization_id=ge_cloud_organization_id,
-        )
-    elif project_config is not None:
+    # First, check for ge_cloud conditions
+
+    config_available = CloudDataContext.is_ge_cloud_config_available(
+        ge_cloud_base_url=ge_cloud_base_url,
+        ge_cloud_access_token=ge_cloud_access_token,
+        ge_cloud_organization_id=ge_cloud_organization_id,
+    )
+
+    # If config available and not explicitly disabled
+    if config_available and ge_cloud_mode != False:
```

Comment:
```suggestion
    if config_available and ge_cloud_mode is not False:
```
I think this is preferred Python.

Reply:
updated in 1ab4ba8d1